Packs/FeedTAXII/Integrations/FeedTAXII2/FeedTAXII2.py

```diff
@@ -48,6 +48,41 @@ def module_test_command(client, limit, fetch_full_feed):
         return_error("Could not connect to server")
 
 
+def filter_indicators(indicators: list, last_run: dict) -> list:
+    """
+    Filter the indicators returned from the taxii server, by taking out indicators that we're ingested in the previous fetch
+    call and their modified date was not updated,
+    """
+    last_indicators = last_run.get("latest_indicators")  # indicators from prev fetch
```

Comment:
Are we storing the entire indicator or just their value and modified date?

Reply:
edited to contain only id and modified date